.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'airbnb-base',
+    'prettier',
+    'eslint-config-prettier',
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  rules: {
+    'import/prefer-default-export': 'none',
+  },
```

Comment:
rule에 prettier에 대한 내용이 없는 것 같아요!
prettierrc의 설정이 실제 적용되고 있는지 확인해보셔야할 것 같습니다

Reply:
제가 eslint 설정을 잘 몰라서 그러는데, prettier 설정은 extends에서 설정하면 되는 거 아니였나요?
prettierrc에서 설정 값을 변경한 후 포맷팅 했을 때 잘 적용되는 것 같거든요.

Reply:
`extends: ["prettier"]` 또는 `extends: ["eslint-config-prettier"]`는 'eslint-config-prettier' 파일의 내용을 가져옵니다(prettier와 겹치는 eslint rule들을 off 해줌)
prettier가 eslint 에러로 잡히도록 하는 플러그인이 eslint-plugin-prettier입니다.
따라서 플러그인에도 추가를 해줘야 하고 실제 rule도 지정을 해줘야합니다.
```json
// .eslintrc
{
    ....

    "plugin": ["prettier"],
    "rules": {
        "prettier/prettier": "error"
    }
}
```
요런식으로요

찾아보니까 .eslintrc에
```json
{
    "extends": ["plugin:prettier/recommende"]
}
```
를 넣어주면 위 설정이 포함된 파일을 알아서 extends 한다고는 하네요 ㅎㅎ
대신 이 경우 임의의 prettier설정이 들어갈 수 있으니 직접 설정하시는 걸 권장합니다.

참고: https://github.com/prettier/eslint-plugin-prettier#readme

Reply:
그리고 제가 놓친 부분이 하나있네요!
```suggestion
  rules: {
    'import/prefer-default-export': 'error',
  },
```
요렇게 해주셔야합니다.

https://eslint.org/docs/user-guide/configuring/rules#rules